lib/controller/controller_test.go

```diff
@@ -38,9 +38,9 @@ import (
 	"k8s.io/client-go/kubernetes/fake"
```

Comment:
please edit `newStorageClass` in this file so that it has a reclaimpolicy. Because we want to pretend it has gone through kubernetes defaulting which would set it to Delete for us.